### ISSUE TITLE
[Question] Multiple Temp Directory Behavior

### ISSUE THREAD

**Issue by broken-gage** [MrPig91/PSChiaPlotter#92]
Hi, 

Sorry if I missed it somewhere. But I watch the tutorial video on the GUI and could find any explanation on the behavior if we have multiple temp drive. 

If I add two temp drive, will the plotter fills the first temp drive until its max allowance then hit the second one? Or it will alter between the two?

P.S. really looking for the phase 1 concurrent limiter. 

Thanks.

**Comment by MrPig91** [comment]
The very first version would chose the temp drive with the most space. It now chooses the the temp drive with the least concurrent processes on it, if tie then the one with the least space. This is true for final drives as well. It will first pick the drive with least number of pending final plots then sort by free space. The real bottleneck is IO throughput, so altering the drives seemed to be the best option (however maybe not always).

As for the phase 1 concurrent limiter. It will probably be on the release after next. I will say that staggering is mostly for avoiding copying files to the same final drive at the same time since this is biggest slow down. Concurrent drives in phase has a lesser impact. I made a video going over this and I first set out to show why to stagger based on phase one but then concluded at the end of the video that the real reason to stagger is for the copying part. However if you are copying to different final drives then staggering matters even less. 

I hope this helps answers your questions :)
